variables.json

```diff
@@ -10,7 +11,7 @@
     "rank9": "<:rank9:1000673940173242378>",
     "rank10": "<:rank10:1000673949761417306>",
     "rankS": "<:rankS:1003216122205708359>",
-    "TOKEN": "NTY0MjU2NzQxNTIxMzU4ODU4.GYBmE2.5kCc33t6vXEA5jHJq8rJ2GQ_IbL8tvfDLMf4Bo",
+    "TOKEN": "NTY0MjU2NzQxNTIxMzU4ODU4.GNn1Ov.8zjNhtwXDVj9Grf2uyaacJdhQTsTf4vxOCxYvc",

```

Comment:
i assume we replaced this token right?

Reply:
ya discord immediately deactivates the bot and DMs me about the token.. so cheese uploaded this around 4:40am or so PST and i was able to get it back up at 5:30am.  However i was just lucky to be awake lol.. I would just replace the token  upon upload with "TOKEN"

Reply:
Should be fixed now.